internal/internal_worker_test.go

```diff
@@ -425,6 +433,13 @@ func createWorkerWithAutoscaler(
 	return createWorkerWithThrottle(t, service, float64(0), WorkerOptions{FeatureFlags: FeatureFlags{PollerAutoScalerEnabled: true}})
 }
 
+func createWorkerWithStrictNonDeterminismOption(
+	t *testing.T,
+	service *workflowservicetest.MockClient,
+) *aggregatedWorker {
+	return createWorkerWithThrottle(t, service, float64(0), WorkerOptions{WorkerBugPorts: WorkerBugPorts{EnableStrictNonDeterminismCheck: true}})
```

Comment:
```suggestion
	return createWorkerWithThrottle(t, service, 0, WorkerOptions{WorkerBugPorts: WorkerBugPorts{EnableStrictNonDeterminismCheck: true}})
```
Explicitly casting a const for a typed arg seems... very strange

Reply:
agreed